### ISSUE TITLE
Featured sites

### ISSUE THREAD

**Issue by Kwpolska** [getnikola/nikola-users#6]
Implementation: https://github.com/getnikola/nikola-users/commit/185427bf92f092d577bf7dd9a8fdb599049b8e1d

@Aeyoun, @ralsina: put ideas here

1. http://legacysurvey.org/ (not on the site yet, would probably need to contact if okay)
2. http://www.stephaniehiga.com/ (145)
3. http://mate-desktop.org/ (184)
4. ???

**Comment by Aeyoun** [comment]
Putting "Powered by Nikola" in Google found these interesting designs:

http://www.damian.oquanta.info/
http://shisaa.jp/

**Comment by Kwpolska** [comment]
This can be done really nicely with nikola-users v3.0.0 ([admin:standard password](https://users.getnikola.com/admin)).

So, what are we going with?  We can add a short description, too.

**Comment by Aeyoun** [comment]
I added two sites as featured.

1. MATE is a high-profile and active project who has done a good job using Nikola. Stephanie
2. Stephanie’s blog is a beautiful showcase of a minimal blog with a healthy number of design and theme tweaks
3. MIST Lab because they’re doing things in space.

We should rotate these every six months or something.

I took the liberty to change their titles and description just a little (deduplication of title in the descriptions) so they would shine just a little brighter at their new positions at the top of the list.
